src/pages/CreateProfile/Activities.tsx

```diff
@@ -0,0 +1,142 @@
+import React from "react";
```

Comment:
Should: resolve errors in console

Warning: Each child in a list should have a unique "key" prop.

Reply:
should be fixed